### ISSUE TITLE
duplicate named cards name convention

### ISSUE THREAD

**Issue by ChrisMoExp** [SWTCG/SWTCG-LACKEY#127]
in Lackey had a couple of cards which had the same name as existing cards. In Lackey we changed them to be named with the set name in braces. I can;t remember now whose idea that was. It seemed fine at the time. A discord user pointed out recently that the cards we renamed in Lackey (of which there were a few for various reasons) were not coming through to decks in Cantina when imported using the Lackey file export. Thats becasue the names aren;t coming up, so i just went through and changed some names in Cantina. But i DIDN'T change these ones with the braces, i notice that the convention in Cantina it use the set acronym without braces. 

SO now we need to agree a convention which works in Cantina and Lackey which is consistent across both. @haec0007 - was it you who favoured the braces? I can't remember. 

here's a list:
AGD - Republic Assault Fleet {AGD}				Republic Assault Fleet (Careful! there's a RAW card called this as well)
AGD - Escape Pod {AGD}						Escape Pod (Careful!!! there is another card from WOTC called this)
AGD - STAP Squad {AGD}						STAP Squad (Careful!!! there is another card from WOTC called this)
AGD - AT-RT Squad							AT RT Squad

**Comment by haec0007** [comment]
Yeah, it was probably my idea to use the braces. That's how I had seen them written out in card lists before, and using braces instead of standard parentheses distinguishes the set abbreviation from a version letter.
